andela/matterhorn-rc
# #152273115 Add option to delete a product.

**Are you submitting to `development`?**

We currently only accept PR's to `development` not `master`

- [x] Description explains the issue/use-case resolved
- [ ] Only contains code directly related to the issue
- [ ] Has tests.
- [ ] Has docs.
- [ ] Passes all tests
- [ ] Has been linted and follows the style guide

#### What does this PR do?
This PR ensures admin can delete products

#### Description of Task to be completed?
 NA

#### How should this be manually tested?
- Pull recent changes into local
- run `npm install` and then run `reaction` from the terminal to start up application
- Log in as an admin and select the product you wish to delete.
- Click the trash icon
- Confirm the delete and voila! it's gone!

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?

#152273115 Add option to delete a product.

#### Screenshots (if appropriate)

<img width="751" alt="screen shot 2017-10-26 at 12 19 05 pm" src="https://user-images.githubusercontent.com/21035946/32050220-eed37460-ba47-11e7-893c-a9f75489eb59.png">



#### Questions:
N/A
